schema/schema.go

```diff
@@ -38,6 +38,25 @@ func (s Schemas) GetByID(id string) (*Schema, error) {
 	return nil, errors.WithStack(herodot.ErrBadRequest.WithReasonf("Unable to find JSON Schema ID: %s", id))
 }
 
+func (s Schemas) Total() int {
+	return len(s)
+}
+
+func (s Schemas) List(page, perPage int) Schemas {
+	if page < 0 {
```

Comment:
The pagination module has a helper for this :)

https://pkg.go.dev/github.com/ory/x/pagination#Index

Reply:
Can you give me a hint on how to do it? I'm a bit confused by the function/how I should use it.